modules/UI/videolayout/RemoteVideo.js

```diff
@@ -230,6 +230,7 @@ RemoteVideo.prototype.remove = function () {
     if (this.container.parentNode) {
         this.container.parentNode.removeChild(this.container);
     }
+    UIUtils.validateTooltips();
```

Comment:
You are right, provided implementation is incomplete, please check [updated version](https://github.com/jitsi/jitsi-meet/pull/939).